src/widgets/Avatar.php

```diff
@@ -59,7 +59,7 @@ public function run()
                         ]
                     ]);
                 } elseif (!empty($meta->avatar)) {
-                    $avatar = Html::img('/avatars/' . $meta->avatar, [
+                    $avatar = Html::img(Yii::getAlias('@web/avatars/') . $meta->avatar, [

```

Comment:
`Html::img()` will call `Yii::getAlias()` itself, so the call here can be skipped.